### PR TITLE
ROSA: update rosa docs for current roles/policies

### PR DESCRIPTION
**Pull request by nermina-redhat**
This applies to `main`, `enterprise-4.9` and `enterprise-4.10`.

This relates to https://issues.redhat.com/browse/OSDOCS-3016. This (static) update lists 4.9 roles, policies and removes 4.7 and 4.8 reference. This PR is an interim step toward automating inclusion of json files from github.

[Preview](https://deploy-preview-40829--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-about-iam-resources)

### CODE REVIEW

**Comment by nermina-redhat** [comment]
@0kashi @arendej please review cc @Tessg22

**Review by bergerhoffer — COMMENTED** [review]
Just two questions, but otherwise LGTM!

File: modules/rosa-sts-account-wide-roles-and-policies.adoc [hunk 1]
```diff
@@ -35,7 +35,7 @@ The account-wide roles and policies are specific to an OpenShift minor release v
             "Effect": "Allow",
             "Principal": {
                 "AWS": [
-                    "arn:aws:iam::710019948333:role/RH-Managed-OpenShift-Installer"
+                    "arn:aws:iam::%{aws_account_id}:role/RH-Managed-OpenShift-Installer"
```
[the remark this answers is not in the document]
Comment (nermina-redhat) [reply modules/rosa-sts-account-wide-roles-and-policies.adoc:38]: this list comes straight from the SRE vaults and will be automatically included in the future. let me know if that answers your question. we're not supposed to modify the file.

[the remark this answers is not in the document]
Comment (bergerhoffer) [reply modules/rosa-sts-account-wide-roles-and-policies.adoc:38]: Yep it does, thanks!

File: modules/rosa-sts-account-wide-roles-and-policies.adoc [hunk 2]
```diff
@@ -970,7 +542,9 @@ The account-wide roles and policies are specific to an OpenShift minor release v
                 "ec2:SearchLocalGatewayRoutes",
                 "ec2:SearchTransitGatewayMulticastGroups",
                 "ec2:SearchTransitGatewayRoutes",
+                "ec2:RunInstances",
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply modules/rosa-sts-account-wide-roles-and-policies.adoc:545]: If that's the case, then this is fine. Thanks!

**Comment by yuwang-RH** [comment]
LGTM.

File: modules/rosa-sts-account-wide-roles-and-policies.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (vkareh) [reply modules/rosa-sts-account-wide-roles-and-policies.adoc:38]: This should be `710019948333` as that's the Service Delivery production account used for running the installer.

**Pull request merged**